### PR TITLE
Updated Autodesk to medium

### PR DESCRIPTION
**Pull request by anestv**
Changed difficulty to medium, because you have to

> Enter your password, then enter a 6-digit code that will be sent to your email and click 'Delete Account'. You can recover the account within 30 days of deleting it. The deletion is completed in 90 days

Source: https://knowledge.autodesk.com/customer-service/account-management/account-profile/account-security/delete-your-autodesk-account and tried it myself

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -540,8 +540,8 @@
     {
         "name": "Autodesk",
         "url": "https://accounts.autodesk.com/Profile/Security",
-        "difficulty": "easy",
-        "notes": "Go to the bottom of the page, click the button 'Account Deletion' and confirm on 'Delete your Autodesk account and data'.",
+        "difficulty": "medium",
+        "notes": "Go to the bottom of the page, click the button 'Account Deletion' and confirm on 'Delete your Autodesk account and data'. Enter your password, then enter a 6-digit code that will be sent to your email and click 'Delete Account'. You can recover the account within 30 days of deleting it. The deletion is completed in 90 days.",
```
Comment (tupaschoal) [inline _data/sites.json:544]: Should the new information replace the old one? Or do you still have to do the first part of the sentence?

Comment (anestv) [reply _data/sites.json:544]: You still have to do the first part too

Comment (tupaschoal) [reply _data/sites.json:544]: Ok, thanks!

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**